src/Chapter2.hs

```diff
@@ -740,7 +749,7 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains n ls = filter (\x -> elem n x) ls
```

Comment:
Now, after you've mastered eta-reduction, you can apply such a technique to other functions as well  🙂 
But your implementation is already great 💯 

Here you have two opportunities for eta-reduction: lambda function and the last argument to `filter`